x/autopilot/keeper/fallback.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v16/x/autopilot/types"
+)
+
+// Stores a fallback address for an outbound transfer
+func (k Keeper) SetTransferFallbackAddress(ctx sdk.Context, channelId string, sequence uint64, address string) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.TransferFallbackAddressPrefix)
+	key := types.GetTransferFallbackAddressKeyPrefix(channelId, sequence)
+	value := []byte(address)
+	store.Set(key, value)
+}
+
+// Removes a fallback address from the store
+// This is used after the ack or timeout for a packet has been received
+func (k Keeper) RemoveTransferFallbackAddress(ctx sdk.Context, channelId string, sequence uint64) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.TransferFallbackAddressPrefix)
+	key := types.GetTransferFallbackAddressKeyPrefix(channelId, sequence)
```

Comment:
Nit: Are `GetPendingSendPacketKey` and `GetTransferFallbackAddressKeyPrefix` both returning Keys, both returning KeyPrefixes, or is one actually returning a Key and the other a KeyPrefix? They look to both be returning KeyPrefixes. 

See counterpart [here](https://github.com/Stride-Labs/stride/blob/main/x/ratelimit/keeper/rate_limit.go#L193).

Reply:
sorry, it should have been `GetTransferFallbackAddressKey`!

topology is:
```go
ctx.KVStore(k.storeKey) → this is a store for just the module
prefix.NewStore(...)    → this is a substore for a particular data structure within the module
key                     → this is a specific byte key
```

Mimicking this structure:
```
{
    "stakeibc": ...
     "autopilot": {
            "fallback_address_prefix_store": {
                    "channel-0,sequence0": "strideXXX", 
                    "channel-0,sequence1": "strideYYY", 
             }
      }
}
```
But really under the hood, there's no nesting, there's just keys and values!

---

(I realize you weren't asking for that explanation / it's probably an overshare but just clarifying so we're on the same page!)

Reply:
Awesome that makes sense and clarifies. Resolving.